src/stylesheets/settings/_mixins.scss

```diff
@@ -416,34 +416,50 @@
 /*--- Avatar Mixins ---*/
 
 @mixin avatar($size: nul) {
-  display: inline-block;
-  width: 100%;
-  vertical-align: top;
-  margin-right: if(
-    $size == "small",
-    $base-spacing-unit-tiny,
-    $base-spacing-unit-small
-  );
-
-  @include media($extra-small) {
+  figure {
     margin-bottom: 0;
-  }
+    margin-right: if(
+      $size == "small",
+      $base-spacing-unit-tiny,
+      $base-spacing-unit-small
+    );
 
-  @if ($size == "small") {
-    max-width: 54px;
-    @include media($small) {
-      max-width: 60px;
+    @if ($size == "small") {
+      max-width: 54px;
```

Comment:
Are these specific, or arbitrary sizes?

Reply:
These are specific sizes as requested by the design team